src/Type/Php/SprintfFunctionDynamicReturnTypeExtension.php

```diff
@@ -118,6 +148,28 @@ public function getTypeFromFunctionCall(
 		return $this->getConstantType($args, $returnType, $functionReflection, $scope);
 	}
 
+	private function getFormatConstantParts(string $format): ?ConstantStringType
+	{
+		$dummyValues = [];
+		for ($i = 0; $i < self::MAX_INTERPOLATION_RETRIES; $i++) {
+			$dummyValues[] = '';
+
+			try {
+				$formatted = @sprintf($format, ...$dummyValues);
```

Comment:
ahh interessting. because of a pre-existing bug, this line results in phpstan errors :).
we are missing `sprintf` can return `false` on php7

https://phpstan.org/r/e0116e22-6029-4e08-9079-4f5128bef6a8

Reply:
declined via https://github.com/phpstan/phpstan-src/pull/3193